pkg/healthcheck/healthcheck_addons.go

```diff
@@ -53,9 +56,15 @@ func (hc *HealthChecker) addOnCategories() []category {
 					check: func(context.Context) error {
 						if grafana, ok := hc.addOns[l5dcharts.GrafanaAddOn]; ok {
 							name, err := getString(grafana, "name")
-							if err != nil {
+							if err != nil && err != errorKeyNotFound {
```

Comment:
It will be helpful for the error message to include which key is missing. Would wrapping `errorKeyNotFound` work?

```
diff --git a/pkg/healthcheck/healthcheck_addons.go b/pkg/healthcheck/healthcheck_addons.go
index c880fd14..a824ca8f 100644
--- a/pkg/healthcheck/healthcheck_addons.go
+++ b/pkg/healthcheck/healthcheck_addons.go
@@ -76,7 +76,7 @@ func (hc *HealthChecker) addOnCategories() []category {
 					check: func(context.Context) error {
 						if grafana, ok := hc.addOns[l5dcharts.GrafanaAddOn]; ok {
 							name, err := getString(grafana, "name")
-							if err != nil && err != errorKeyNotFound {
+							if err != nil && errors.Is(err, errorKeyNotFound) {
 								return err
 							}
 
@@ -283,7 +283,7 @@ func getString(i interface{}, k string) (string, error) {
 
 	v, ok := m[k]
 	if !ok {
-		return "", errorKeyNotFound
+		return "", fmt.Errorf("%w: %s", errorkeyNotFound, k)
 	}
 
 	res, ok := v.(string)
```

Reply:
@ihcsim I tried the above wrapping, but then it wasn't recognised as a errorKeyNotFound for us to compare. :thinking: 

Also, Updated the code to use `errors.Is` instead of `==`